src/Microsoft.ComponentDetection.Orchestrator/Commands/ScanSettings.cs

```diff
@@ -0,0 +1,63 @@
+namespace Microsoft.ComponentDetection.Orchestrator.Commands;
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using Microsoft.ComponentDetection.Orchestrator.Extensions;
+using Spectre.Console.Cli;
+
+/// <summary>
+/// Settings for the Scan command.
+/// </summary>
+public class ScanSettings : BaseSettings
+{
+    [CommandOption("--DirectoryExclusionList")]
+    [Description("Filters out specific directories following a minimatch pattern.")]
+    [TypeConverter(typeof(SemicolonDelimitedConverter))]
+    public IEnumerable<string> DirectoryExclusionList { get; set; }
+
+    [CommandOption("--IgnoreDirectories", IsHidden = true)]
+    [Description(
+        "Filters out specific directories, providing individual directory paths separated by semicolon. Obsolete in favor of DirectoryExclusionList")]
+    [TypeConverter(typeof(CommaDelimitedConverter))]
```

Comment:
docs say `paths separated by semicolon` but this uses a `CommaDelimitedConverter`.